### PR TITLE
Burger menu design 

### PR DESCRIPTION
**Pull request by tawfiknasser**
**in this PR** we imported the react-burger-menu package so we could use ready component and save time ,
it could be changed later ,
the burger menu is working and it show three links when clicking on the bar .
this links now are '/' later when we finish the authentication we will add functionality to these links.

### CODE REVIEW

**Comment by Karyum** [comment]
@tawfiknasser umm  what ? everyone approved but who is the qa/

**Comment by shiryz** [comment]
@Karyum I'm back to QA frontend, don't take my contributions

File: client/src/components/loginViewContainer.js [hunk 1]
```diff
@@ -1,6 +1,5 @@
 import React, { Component } from "react";
 import Login from "./LoginComponent/login";
-import style from "./LoginComponent/login.css";
 
 const LoginBody = () => {
   return <Login />;
```
Comment (shiryz) [inline client/src/components/loginViewContainer.js]: this seems redundant now that you only use it as a container, sorry I didn't catch this before
Unless this component will have more stuff in it

Comment (shiryz) [reply client/src/components/loginViewContainer.js]: I'm talking about importing Login and then exporting the same thing wrapped in LoginBody

**Comment by shiryz** [comment]
@tawfiknasser you need to reference the issue you're working on in the description

File: client/src/components/loginViewContainer.js (hunk 1 above)
Comment (yoosefkh) [reply client/src/components/loginViewContainer.js]: we resolved this in the next PR

**Pull request merged**